[docs] Cloud docs (#9920) (#9964)
yugabyte/yugabyte-db: docs/content/latest/yugabyte-cloud/cloud-clusters/overview.md

- Operations/sec - Shows the read and write operations on the cluster over time.
- Average Latency - Shows the average amount of time in milliseconds taken for read and write operations.
- CPU Usage - Shows the percentage of CPU use for the cluster.
- Disk Usage - Shows the amount of disk space provisioned for and used by the cluster.

> Metrics can be displayed over the past hour, 6 hours, 12 hours, 24 hours, or 7 days.

Reviewer: ```suggestion
You can show metrics for the past hour, 6 hours, 12 hours, 24 hours, or 7 days.
```
